aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -0,0 +1,158 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    aggregator_v2::{
+        check, initialize, materialize, materialize_and_try_add, materialize_and_try_sub, new,
+        try_add, try_add_and_materialize, try_sub, try_sub_add, try_sub_and_materialize,
+    },
+    assert_abort, assert_success,
+    tests::common,
+    MoveHarness,
+};
+use aptos_language_e2e_tests::account::Account;
+use aptos_types::transaction::SignedTransaction;
+
+fn setup() -> (MoveHarness, Account) {
+    initialize(common::test_dir_path("aggregator.data/pack"))
```

Comment:
Btw your set-up builds both V1 and V2 Move files for both V1 and V2 tests, so x2 times more than needed, I think... maybe having it in a separate package is better, or you can merge tests alternatively.